crates/node-builder/src/builder.rs

```diff
@@ -461,7 +461,7 @@ where
 
         let genesis_hash = init_genesis(provider_factory.clone())?;
 
-        info!(target: "reth::cli", "{}",config.chain.display_hardforks());
+        info!(target: "reth::cli", "\n\n{}", config.chain.display_hardforks());
```

Comment:
I'd like to not have empty empty lines
```suggestion
        info!(target: "reth::cli", "\n{}", config.chain.display_hardforks());
```

and ideally we also prevent trailing new line for the last element here

https://github.com/paradigmxyz/reth/blob/e88b16e6e784da7db5fb9cdae3ee7868c3b0790b/crates/primitives/src/chain/spec.rs#L1535-L1537

Reply:
@mattsse could you please give more tips about the tailing newline issue, and just curious why there is a new line after with-merge and post-merge blocks, but no newline after the pre-merge?

Reply:
the bug is here:

https://github.com/paradigmxyz/reth/blob/e88b16e6e784da7db5fb9cdae3ee7868c3b0790b/crates/primitives/src/chain/spec.rs#L1457-L1457

this should just be write!